src/server/object_services/schemas/object_md_indexes.js

```diff
@@ -145,4 +144,21 @@ module.exports = [
         }
     },
 
+    //////////////////////
+    // OBJECT RECLAIMER //
+    //////////////////////
+
+    {
+        fields: {
+            deleted: 1,
```

Comment:
I remember there was some issue with having two indexes that have the same fields but differ only in their partialFilterExpression - see here:
https://github.com/noobaa/noobaa-core/blob/f6f10dcac7457c136bf858d0d07d946f54ba057d/src/server/object_services/schemas/object_md_indexes.js#L14-L16
Is that working for you?

Reply:
it doesn't added reclaimed to the fields list (will always be null)